release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -437,7 +437,7 @@ The Machine Config Operator (MCO) now updates the affected nodes alphabetically
 For more information, see xref:../post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
 
 [id="ocp-4-11-mco-signer-ca-alert"]
-==== New alert if MCO cannot update the signer CA on a paused node
+==== Enhanced notification for paused Machine Config Pools upon certificate renewal
 
 You will now receive alerts in the Alerting UI of the {product-title} web console if the MCO attempts to renew an expired `kube-apiserver-to-kubelet-signer` CA certificate on a Machine Config Pool (MCP) that is paused. If the MCPs are paused, the MCO cannot push the newly rotated certificates to those nodes, which can result in failures.
```

Comment:
* If "Alerting" is a UI element or label, it should be bold, IIRC. 
* Should Machine Config Pool be "machine config pool"? That's how we have it elsewhere in the docs per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

Reply:
@maxwelldb Thanks for the review. 

Alerting UI is not bolded in the docs.
Maching config pool is lower-case, yes.